### PR TITLE
Adding a basic project structure

### PR DESCRIPTION
**Pull request by mythrilium**
Added contributing guidelines
Added a naming convention for ease of use

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Frequent Asked Questions
+
+Comonly asked questions about the project and hobby.
```
Comment (jMavarez) [inline FAQ.md]:
```suggestion
Commonly asked questions about the project and hobby.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+In case of the resource being a QMK keymap, please use the following naming convention:
+
+```text
+<keyboard>-<layout>-<username>.json
+```
+
+If your username contains hyphens, please replace them with underscores. Try to use the same username that you use in the QMK configurator.
+
+Example: `planck/rev6/myth.json`
```
Comment (jMavarez) [inline CONTRIBUTING.md]:
```suggestion
Example: `planck-rev6-myth.json`
```

Comment (luisleoncasas) [reply CONTRIBUTING.md]: Fixed

File: FAQ.md (hunk 1 above)
Comment (luisleoncasas) [reply FAQ.md]: Fixed

Comment (mythrilium) [reply FAQ.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (mythrilium) [reply CONTRIBUTING.md]: fixed

**Review by jMavarez — APPROVED** [review]

**Pull request merged**